From 74aa9c38f5eb9d5de90c7c9e68f66c5554715076 Mon Sep 17 00:00:00 2001
From: David Wu <david.wu@rock-chips.com>
Date: Wed, 14 Nov 2018 17:46:01 +0800
Subject: [PATCH] arm64: dts: rockchip: Add usbdrd3's compatible
 "rockchip,rk3399-dwc3" for rk1808

We will use the dwc3-rockchip driver for rk1808, this patch is prepared
for it.

Change-Id: I7ca8baefd26ea6c67140b757c47e14625cfed609
Signed-off-by: David Wu <david.wu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808.dtsi | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808.dtsi b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
index b26262a7a3d5..52ac30796ced 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
@@ -212,7 +212,7 @@
 	};
 
 	usbdrd3: usb {
-		compatible = "rockchip,rk1808-dwc3";
+		compatible = "rockchip,rk1808-dwc3", "rockchip,rk3399-dwc3";
 		clocks = <&cru SCLK_USB3_OTG0_REF>, <&cru ACLK_USB3OTG>,
 			 <&cru SCLK_USB3_OTG0_SUSPEND>;
 		clock-names = "ref_clk", "bus_clk",
@@ -220,6 +220,8 @@
 		assigned-clocks = <&cru SCLK_USB3_OTG0_SUSPEND>;
 		assigned-clock-rates = <24000000>;
 		power-domains = <&power RK1808_PD_PCIE>;
+		resets = <&cru SRST_USB3_OTG_A>;
+		reset-names = "usb3-otg";
 		#address-cells = <2>;
 		#size-cells = <2>;
 		ranges;
-- 
2.35.3

